### PR TITLE
ymlの追加とリポジトリのParamを変更をしました

### PR DESCRIPTION
**Pull request by SekiMoemi**

### CODE REVIEW

File: src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java [hunk 1]
```diff
@@ -21,51 +21,52 @@
...
-	 */
-	public List<Item> findAll() {
-		String sql = "Select id, name, description, price_m, price_l, image_path, deleted from items;";
-		List<Item> itemList = template.query(sql, ITEM_ROW_MAPPER);
-		return itemList;
-	}
+  /**
+   * アイテム（おもちゃ）覧情報を取得します.
+   * 
+   * @return 全おもちゃのリスト
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java:35]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.RequireEmptyLineBeforeBlockTagGroupCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Javadoc tag '@return' should be preceded with an empty line.

File: src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java [hunk 2]
```diff
@@ -21,51 +21,52 @@
...
-	 * 
-	 * @param name おもちゃの名前
-	 * @return 検索された全おもちゃのリスト
-	 */
-	public List<Item> findByName(String name) {
-		String sql = "Select id, name, description, price_m, price_l, image_path, deleted from items where name LIKE :name ORDER BY id;";
-		SqlParameterSource param = new MapSqlParameterSource().addValue("name", "%" + name + "%");
+  /**
+   * 名前でアイテム（おもちゃ）を曖昧検索する.
+   * @param name おもちゃの名前
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java:45]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.RequireEmptyLineBeforeBlockTagGroupCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Javadoc tag '@param' should be preceded with an empty line.

File: src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java [hunk 3]
```diff
@@ -21,51 +21,52 @@
...
+   * @return 検索された全おもちゃのリスト
+   */
+  public List<Item> findByName(String name) {
+    String sql =
+        "Select id, name, description, price_m, price_l, image_path, deleted from items where name LIKE :name ORDER BY id;";
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java:50]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 124).

File: src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java [hunk 4]
```diff
@@ -21,51 +21,52 @@
...
-	 */
-	public Item load(Integer id) {
-		String sql = "SELECT id, name, description, price_m, price_l, image_path, deleted FROM items WHERE id=:id;";
-		SqlParameterSource param = new MapSqlParameterSource().addValue("id", id);
-		Item item = template.queryForObject(sql, param, ITEM_ROW_MAPPER);
-		return item;
+  /**
+   * 商品IDによってアイテム（おもちゃ）を取得するメソッド.
+   * 
+   @param id 商品ID
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java:60]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.RequireEmptyLineBeforeBlockTagGroupCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Javadoc tag '@param' should be preceded with an empty line.

File: src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java [hunk 5]
```diff
@@ -21,51 +21,52 @@
...
+   * @return アイテム（おもちゃ）情報
+   */
+  public Item load(Integer id) {
+    String sql =
+        "SELECT id, name, description, price_m, price_l, image_path, deleted FROM items WHERE id=:id;";
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java:65]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 103).

**Review by yamada-kensyu — APPROVED** [review]
承認します

**Pull request merged**